src/select_GCMs.R

```diff
@@ -9,267 +9,301 @@
-                           delim = ".",
-                           cols_remove  = FALSE) %>% 
-      filter(!GCM_only %in% low_skill_models$GCM)
-    
-    # assign climate futures
-    
-    #### Set limits for CF classification
-    Pr0 <-  as.numeric(quantile(future_means$delta_pr, 0))
-    Pr25 <-  as.numeric(quantile(future_means$delta_pr, 0.25))
-    PrAvg <-  mean(future_means$delta_pr)
-    Pr75 <-  as.numeric(quantile(future_means$delta_pr, 0.75))
-    Pr100 <-  as.numeric(quantile(future_means$delta_pr, 1))
-    Tavg0 <-  as.numeric(quantile(future_means$delta_tavg, 0))
-    Tavg25 <-  as.numeric(quantile(future_means$delta_tavg, 0.25)) 
-    Tavg <-  mean(future_means$delta_tavg)
-    Tavg75 <-  as.numeric(quantile(future_means$delta_tavg, 0.75))
-    Tavg100 <-  as.numeric(quantile(future_means$delta_tavg, 1))
-    
-    # CF assignment
-    future_means <- future_means %>%
-      # designate climate future classification based on cf limits
-      mutate(
-        CF = case_when(
-          delta_tavg < Tavg &
-            delta_pr > Pr75 |
-            delta_tavg < Tavg25 & delta_pr > PrAvg ~ "Warm Wet",
-          delta_tavg > Tavg &
-            delta_pr > Pr75 |
-            delta_tavg > Tavg75 & delta_pr > PrAvg ~ "Hot Wet",
-          delta_tavg > Tavg25 &
-            delta_tavg < Tavg75 &
-            delta_pr > Pr25 & delta_pr < Pr75 ~ "Central",
-          delta_tavg < Tavg &
-            delta_pr < Pr25 |
-            delta_tavg < Tavg25 & delta_pr < PrAvg ~ "Warm Dry",
-          delta_tavg > Tavg &
-            delta_pr < Pr25 |
-            delta_tavg > Tavg75 & delta_pr < PrAvg ~ "Hot Dry"
+    # create empty list to fill
+    selected_gcms <- vector("list", length = length(future_dfs))
+    
+    for (i in 1:length(future_dfs)) {
+      # clean data
+      ## future data
+      future_all <- future_dfs[[i]] %>%
+        dplyr::rename(
+          precip_in = `Precip (in)`,
+          tmin_f = `Tmin (F)`,
+          tmax_f = `Tmax (F)`,
+          rhmax = `RHmax (%)`,
+          rhmin = `RHmin(%)`,
+          tavg_f = `Tavg (F)`
+        ) %>%
+        mutate(year = format(Date, "%Y"),
+               #VPD = VPD(tmin_f, tmax_f, rhmin, rhmax), # do we need vapor pressure??
+               DOY = yday(Date)) %>%
+        filter(year %in% future_range)
+      
+      
+      ## historic data
+      historic_all <- historic_dfs[[i]] %>%
+        dplyr::rename(
+          precip_in = `Precip (in)`,
+          tmin_f = `Tmin (F)`,
+          tmax_f = `Tmax (F)`,
+          rhmax = `RHmax (%)`,
+          rhmin = `RHmin(%)`,
+          tavg_f = `Tavg (F)`
+        ) %>%
+        mutate(year = format(Date, "%Y"),
+               #VPD = VPD(tmin_f, tmax_f, rhmin, rhmax),
+               DOY = yday(Date)) %>%
+        filter(year %in% historic_range)
+      
+      
+      
+      
+      # calculate deltas
+      
+      # baseline means from historic data
+      baseline <- historic_all %>%
+        summarise(
```

Comment:
Summarize is another function that regularly errors out due to package conflicts. Add dplyr:: if possible before all occurrences (again on lines 108 and 121).